crates/db_schema/src/impls/community.rs

```diff
@@ -194,6 +194,38 @@ impl DeleteableOrRemoveable for Community {
   }
 }
 
+pub enum CollectionType {
+  Moderators,
+  Featured,
+}
+
+impl Community {
```

Comment:
These only need the `Community`, not the `CommunityView`, correct? Because if they have to refetch the view, then it'd be better to add these into the `CommunityView` impls.

Reply:
Federation code doesnt use any of the *View types.